lib/msf/base/serializer/opt_conditions.rb

```diff
@@ -0,0 +1,54 @@
+# -*- coding: binary -*-
+module Msf
+  module Serializer
+    module OptConditions
+
+      # Check a condition's result
+      def self.eval_condition(left_value, operator, right_value)
+        result = true
+        case operator
+        when "=="
+          result = left_value == right_value
+        when "!="
+          result = left_value != right_value
+        when "in"
+          result = right_value.include?(left_value)
+        when "nin"
+          result = right_value.include?(left_value)
+          result = !result
+        end
+
+        result
+      end
+
+      # Check an OPTION conditions. This function supports
+      # dump_options()
+      def self.opt_condition_checked(condition, mod, opt)
```

Comment:
Could you please change this to:
```
def self.show_option(mod, option)
```

That would:
1. Make it a little easier to read with the usage being `next unless Msf::Serializer::OptConditions.show_option(mod, opt)`
1. Would make the signature cleaner, you can access the condition directly from `opt.conditions`.

Reply:
Done!